### PR TITLE
05.08 release notes

### PR DESCRIPTION
**Pull request by lzdanski**
- Adds release notes
- Adds info WRT https://github.com/astronomer/astro/pull/20756

### CODE REVIEW

File: astro/create-dedicated-cluster.md [hunk 1]
```diff
@@ -29,43 +29,53 @@ Dedicated clusters offer the self-service convenience of a fully managed service
...
+    - **VPC Subnet Range**: Specify the range used by nodes in your GKE cluster (Default: `172.20.0.0/22`). Astro uses this range to make private connections to your target data sources.
+
+4. (Optional) Configure the following **Advanced Configuration** details about your cluster for Private Networking.
+
+    Astro uses source network address translation (SNAT) that performs many-to-one IP address translations for connections to your data sources, to minimize the risk and concern with IP overlap and exhaustion. When using private networking, like VPN or VPC Peering, between Astro and your target data sources, your target data sources see connections from the default **VPC Subnet Range**. If you're using private connections, confirm that the following **Advanced Configuration** network ranges do not overlap with your target data source networks.
+
+    - **Pod Subnet Range**: Specify the range used by GKE Pods (Default: `100.64.0.0/16`).
+    - **Service Subnet Range**: Specify the range used by nodes in your GKE cluster (Default: `100.65.0.0/22`).
+    - **Service Peering Range**: Specify the range used by Private Service connections ( Default: `100.66.0.0/21`)
```
Comment (TJaniF) [inline astro/create-dedicated-cluster.md:65]:
```suggestion
    - **Service Peering Range**: Specify the range used by Private Service connections (Default: `100.66.0.0/21`)
```

**Review by TJaniF — APPROVED** [review]
LGTM from a copy edit perspective. I don't know enough about VPC peering to review the content for accuracy. 😅

**Comment by chomipi88** [comment]
@lzdanski While the code has been live for some time, we will finally be cutting over to the new image registry backend this week as part of a larger reliability project. This is an extension to the previously released [registry cache](https://deploy-preview-3711--gallant-neumann-56599d.netlify.app/astro/release-notes#new-deployment-registry-cache-to-improve-resiliency).
See attached doc for details of the cutover to the new registry backend.0
[Migration of Astro Image Registries.pdf](https://github.com/astronomer/docs/files/15238749/Migration.of.Astro.Image.Registries.pdf)

File: astro/create-dedicated-cluster.md [hunk 2]
```diff
@@ -29,43 +29,56 @@ Dedicated clusters offer the self-service convenience of a fully managed service
...
-    - **Service VPC Peering**: Specify the range used by Private Service connections (Default: `172.23.0.0/20`).
-   
-4. Click **Create cluster**. After Astro finishes creating the cluster, users in your Organization can select the cluster when they [create a Deployment](create-deployment.md). 
+    - **VPC Subnet Range**: Specify the range used by nodes in your GKE cluster (Default: `172.20.0.0/22`). Astro uses this range to make private connections to your target data sources.
+
+4. (Optional) Configure the following **Advanced Configuration** details about your cluster for Private Networking.
+
+    Astro uses source network address translation (SNAT) that performs many-to-one IP address translations for connections to your data sources, to minimize the risk and concern with IP overlap and exhaustion. When using private networking, like VPN or VPC Peering, between Astro and your target data sources, your target data sources see connections from the default **VPC Subnet Range**. If you're using private connections, confirm that the following **Advanced Configuration** network ranges do not overlap with your target data source networks.
```
Comment (chomipi88) [inline astro/create-dedicated-cluster.md:61]:
```suggestion
    Astro uses source network address translation (SNAT) that performs many-to-one IP address translations for connections to your data sources and defaults secondary ranges to RFC 6598 address space (non-standard Private IP addresses), to minimize the risk and concern with IP overlap and exhaustion. When using private networking, like VPN or VPC Peering, between Astro and your target data sources, your target data sources see connections from the default **VPC Subnet Range**. If you're using private connections, confirm that the following **Advanced Configuration** network ranges do not overlap with your target data source networks.
```

File: astro/release-notes.md [hunk 3]
```diff
@@ -25,6 +25,26 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
 
 <!-- ALL LINKS TO INTERNAL DOCS MUST BE COMPLETE URLS INCLUDING HTTPS. Otherwise the links will break in RSS. -->
 
+## May 7, 2024
+
+### Updates to address ranges for dedicated clusters on Google Cloud Provider
+
+Astro on GCP Dedicated Clusters uses source network address translation (SNAT) that performs many-to-one IP address translations for connections to your data sources, to minimize the risk and concern with IP overlap and exhaustion. Your target data sources will see connections from Astro using the VPC Subnet Range when using private networking, like VPC Peering or VPN. If you want to configure private connectivity, ensure the default subnet and peering ranges do not overlap with your target data source network when you're creating your dedicated cluster. See [Create a dedicated Astro cluster](https://docs.astronomer.io/astro/create-dedicated-cluster?tab=gcp#create-a-cluster) for more details.
```
Comment (chomipi88) [inline astro/release-notes.md:32]:
```suggestion
Astro on GCP Dedicated Clusters uses source network address translation (SNAT) that performs many-to-one IP address translations for connections to your data sources and defaults secondary ranges to RFC 6598 address space (non-standard Private IP addresses), to minimize the risk and concern with IP overlap and exhaustion. Your target data sources will see connections from Astro using the VPC Subnet Range when using private networking, like VPC Peering or VPN. If you want to configure private connectivity, ensure the default subnet and peering ranges do not overlap with your target data source network when you're creating your dedicated cluster. See [Create a dedicated Astro cluster](https://docs.astronomer.io/astro/create-dedicated-cluster?tab=gcp#create-a-cluster) for more details.
```

**Review by chomipi88 — APPROVED** [review]

File: astro/release-notes.md [hunk 4]
```diff
@@ -25,6 +25,26 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
 
 <!-- ALL LINKS TO INTERNAL DOCS MUST BE COMPLETE URLS INCLUDING HTTPS. Otherwise the links will break in RSS. -->
 
+## May 7, 2024
```
Comment (lzdanski) [inline astro/release-notes.md:28]:
```suggestion
## May 8, 2024
```

File: astro/release-notes.md [hunk 5]
```diff
@@ -25,6 +25,26 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+
+### Improvements to Astro performance
+
+As part of continued investment in the reliability, performance, and scalability of Astro, Astronomer is embarking on a migration of our public and private image registries. Astro Runtime clusters will benefit from more performant, globally distributed and geo-replicated image registries, with built-in registry resilience if a regional outage occurs. This is in addition to the previous release of the registry cache local to every Astro cluster.
+
+No end user or task runtime change or impact is expected as part of the backend cutover during the week of May 6th, 2024
```
Comment (lzdanski) [inline astro/release-notes.md:38]:
```suggestion
No end user or task runtime change or impact is expected as part of the backend cutover during the week of May 6th, 2024.
```

File: astro/release-notes.md [hunk 6]
```diff
@@ -25,6 +25,26 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+
+### Improvements to Astro performance
+
+As part of continued investment in the reliability, performance, and scalability of Astro, Astronomer is embarking on a migration of our public and private image registries. Astro Runtime clusters will benefit from more performant, globally distributed and geo-replicated image registries, with built-in registry resilience if a regional outage occurs. This is in addition to the previous release of the registry cache local to every Astro cluster.
```
Comment (lzdanski) [inline astro/release-notes.md:36]:
```suggestion
As part of continued investment in the reliability, performance, and scalability of Astro, Astronomer is embarking on a migration of public and private image registries. Astro Runtime clusters will benefit from more performant, globally distributed and geo-replicated image registries, with built-in registry resilience if a regional outage occurs. This is in addition to the previous release of the registry cache local to every Astro cluster.
```

File: astro/runtime-version-lifecycle-policy.mdx [hunk 7]
```diff
@@ -57,6 +57,10 @@ Astronomer will not interrupt service for Deployments running Astro Runtime vers
 
 Maintenance is discontinued the last day of the month for a given version. For example, if the maintenance window for a version of Astro Runtime is January - June of a given year, that version will be maintained by Astronomer until the last day of June.
 
+### Restricted runtime versions
+
+There are some restricted versions of the Astro runtime that have known bugs, and cannot be used, even if you have [Running a deprecated Astro Runtime version](https://docs.astronomer.io/astro/upgrade-runtime#run-a-deprecated-astro-runtime-version) enabled. These invalid versions of the Astro Runtime have a status `"yanked":true` if you query `https://updates.astronomer.io/astronomer-runtime` with a description about why the version is unavailable.
```
Comment (lzdanski) [inline astro/runtime-version-lifecycle-policy.mdx:62]:
```suggestion
There are some restricted versions of the Astro runtime that have known bugs, and cannot be used, even if you have [Running a deprecated Astro Runtime version](https://docs.astronomer.io/astro/upgrade-runtime#run-a-deprecated-astro-runtime-version) enabled. These invalid versions of the Astro Runtime have a status `"yanked":true` if you query `https://updates.astronomer.io/astronomer-runtime` and include a description about why the version is unavailable.
```

**Review by chomipi88 — APPROVED** [review]

File: astro/release-notes.md [hunk 8]
```diff
@@ -25,6 +25,26 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+
+No end user or task runtime change or impact is expected as part of the backend cutover during the week of May 6th, 2024
+
+### Additional improvements
+
+- You can no longer create Deployments using Astro Runtime versions marked as `yanked` in `https://updates.astronomer.io/astronomer-runtime`, even if your Organization has enabled creating Deployments with deprecated Runtime versions. These versions of the Astro Runtime have known issues and should not be used.
```
Comment (lzdanski) [inline astro/release-notes.md:42]:
```suggestion
- You can no longer create Deployments using Astro Runtime versions marked as `yanked` in `https://updates.astronomer.io/astronomer-runtime`, even if your Organization has enabled creating Deployments with deprecated Runtime versions. These versions of the Astro Runtime have known issues and should not be used. For more information, see [Restricted Runtime Versions](https://docs.astronomer.io/astro/runtime-version-lifecycle-policy#restricted-runtime-versions).
```

**Pull request merged**